sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -283,6 +286,20 @@ public AutoConfiguredOpenTelemetrySdkBuilder addMetricExporterCustomizer(
     return this;
   }
 
+  /**
+   * Adds a {@link BiFunction} to invoke with the autoconfigured {@link MetricReader} to allow
+   * customization. The return value of the {@link BiFunction} will replace the passed-in argument.
+   *
+   * <p>Multiple calls will execute the customizers in order.
+   */
+  @Override
+  public AutoConfigurationCustomizer addMetricReaderCustomizer(
```

Comment:
Can you change the response type to be `AutoConfiguredOpenTelemetrySdkBuilder`? #6248 fixes another instance of this and we don't want to extend the issue further.

Reply:
@jack-berg Changed

Reply:
Looks like you need to re-run `./gradlew japicmp`